plugin/src/main/java/io/jenkins/plugins/casc/yaml/MergeStrategyAction.java

```diff
@@ -0,0 +1,54 @@
+package io.jenkins.plugins.casc.yaml;
+
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.model.Api;
+import hudson.model.UnprotectedRootAction;
+import hudson.util.HttpResponses;
+import javax.annotation.CheckForNull;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONArray;
+import org.jenkinsci.Symbol;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.export.ExportedBean;
+
+@Extension
+@Symbol("cascMergeStrategy")
+@ExportedBean
+public class MergeStrategyAction implements UnprotectedRootAction {
```

Comment:
What is this class for?

Reply:
This is an API for getting the existing strategies. I've added the test cases.

Reply:
why do you need it?

Reply:
I integrate Jenkins as a CI engine in [this project](https://github.com/kubesphere/ks-devops). I wish I can get the supported strategies via API.